scripts/benchmark.js

```diff
@@ -20,7 +20,6 @@ const butternut = require("butternut");
 
 const ASSETS_DIR = path.join(__dirname, "benchmark_cache");
 const DEFAULT_ASSETS = {
-  "react.js": "https://unpkg.com/react/dist/react.js", // 120 kb
```

Comment:
we would like to keep react as a dep here since we use it for smoke tests. 

So please update with the new URL instead of removing it. You can find the relevant code here https://unpkg.com/react@16.3.1/

Reply:
@vigneshshanmugam should i update it with  `react.development.js ` or `react.production.js`  ??

Reply:
use development.js